### ISSUE TITLE
Bug: XState Inspect 0.8.0 doesn't work with XState 5.0.0-beta9

### ISSUE THREAD

**Issue by skyberd** [statelyai/xstate#4010]
### Description

You cannot inspect state machines that use v5 of XState using @xstate.inspect v0.8.0
I think a new beta release of xstate inspect is required.

### Expected result

I expect inspect to work

### Actual result

When starting my node server I get the following error:

```
(this will be run only when your dependencies or config have changed)
✘ [ERROR] No matching export in "node_modules/xstate/dist/xstate.esm.js" for import "toEventObject"

    node_modules/@xstate/inspect/es/browser.js:2:34:
      2 │ import { interpret, toSCXMLEvent, toEventObject, toObserver, toActorRef } from 'xstate';
        ╵                                   ~~~~~~~~~~~~~

✘ [ERROR] No matching export in "node_modules/xstate/dist/xstate.esm.js" for import "toActorRef"

    node_modules/@xstate/inspect/es/browser.js:2:61:
      2 │ import { interpret, toSCXMLEvent, toEventObject, toObserver, toActorRef } from 'xstate';
        ╵                                                              ~~~~~~~~~~

```

### Reproduction

This cannot be reproduced in the code sandbox

**Comment by Andarist** [comment]
> I think a new beta release of xstate inspect is required.

Yes, this. v5 is a work in progress though and the tooling around it will be released once it settles a little bit more.

**Comment by inverted-capital** [comment]
WIP is happening here: https://github.com/statelyai/xstate/pull/4082

**Comment by davidkpiano** [comment]
> WIP is happening here: #4082

Thanks! This is very close to being merged in. Just needs some cleanup and the @Andarist stamp of approval.

**Comment by ntvinhit** [comment]
> WIP is happening here: https://github.com/statelyai/xstate/pull/4082

Looks like it doesn't work with the current official v5 :(

**Comment by davidkpiano** [comment]
> > WIP is happening here: #4082
> 
> Looks like it doesn't work with the current official v5 :(

Still experimental, but I recommend using `@statelyai/inspect` instead, which works with XState v5 and has an API to work with anything else (even non-XState code): 

https://github.com/statelyai/inspect

**Comment by ntvinhit** [comment]
I tried with @statelyai/inspect
But it just open a new tab with src: `https://stately.ai/inspect`.

And it shows "404 Page Not Found".

So I assume it won't work event with new library @davidkpiano

**Comment by davidkpiano** [comment]
> I tried with @statelyai/inspect
> But it just open a new tab with src: `https://stately.ai/inspect`.
> 
> And it shows "404 Page Not Found".
> 
> So I assume it won't work event with new library @davidkpiano 
> 

For now, set the `url` to https://stately.ai/registry/inspect

**Comment by ntvinhit** [comment]
>For now, set the url to https://stately.ai/registry/inspect
Thank you, it works now but the events doesn't reflect to the inspector.
But it's absolutely okay for me because I just want to view the statechart.

FYI:
I tried to use the inspector in iframe mode but looks like I can't login (it force me to login to see the content).
I tried to log in with Google but of course we cant do that because Google has `X-Frame-Options: deny`.

I tried to `Reset password` but after I clicked the Reset link, it login automatically instead of letting me to update the password.

=> I have to use the windows mode.
